src/backend/web/routes/index.js

```diff
@@ -17,7 +17,7 @@ const user = require('./user');
 const query = require('./query');
 
 const router = express.Router();
-
+const { FRONTEND } = process.env.FRONTEND;
```

Comment:
In the rest of this code, we seem to be doing `process.env.FRONTEND` inline, instead of pulling things out.  Can you match that below?

Reply:
Careful here: using [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring), you have to remove the name of the variable in the rhs.

```ts
const { FRONTEND } = process.env;
```

Reply:
Will do! Everything should be fixed along with the rest of your suggestions